customizations/tests.py

```diff
@@ -88,7 +88,7 @@ def test_post(self, mock_upload_file, mock_webhook, mock_token):
                 u'footer_description': [u'asdasd'],
                 u'show_event_sequence': [u'on'],
                 u'create_customization': [u'Create'],
-                u'select_design_template': [u'DESIGN 1'],
+                u'select_design_template': [u'1'],
```

Comment:
This field was eliminated, it should not appear in the test, right?

Reply:
This field continues in the form, the difference is that you should send the number of the template for example in this case the test simulates that we choose the first one